libs/eo/landing-page/shell/src/lib/eo-landing-page-header.component.ts

```diff
@@ -57,12 +57,14 @@ const selector = 'eo-landing-page-header';
           background: var(--watt-color-primary);
           color: var(--watt-color-primary-contrast);
 
-          // This is equivalent to a normal size watt button
+          // This is equivalent to a normal size watt button width
           min-width: 6.25rem;
-          height: 2.75rem;
+          height: calc(5 * var(--watt-space-s)); // 40px
```

Comment:
suggestion: (1) Use multiples of the XS space unit unless the size is relative to another component. (2) Let's refrain from using comments with values that can get out of sync.
```suggestion
          height: calc(10 * var(--watt-space-xs));
```